tests/fixtures/entities.py

```diff
@@ -7,22 +7,33 @@
 from pontoon.base.models import Entity
 
 
-@pytest.fixture
-def entity_factory(factory):
-
-    def instance_attrs(instance, i):
-        if not instance.string:
-            instance.string = "Entity %s" % i
-
-    return functools.partial(
-        factory, model=Entity, instance_attrs=instance_attrs)
-
-
 @pytest.fixture
 def entity0(resource0):
+    """Entity 0"""
     return Entity.objects.get(resource=resource0, string="entity0")
 
 
 @pytest.fixture
 def entity1(resource1):
+    """Entity 1"""
     return Entity.objects.get(resource=resource1, string="entity1")
+
+
+@pytest.fixture
+def entity_factory(factory):
+    """Entity factory
+    create entities in a hurry!
+
+    Provides an entity factory function that accepts the following args:
+
+    :param `batch`: number of entities to instantiate, defaults to len of
+        `batch_kwargs` or 1
+    :param `batch_kwargs`: a list of kwargs to instantiate the entities
```

Comment:
This is not consistent with what we already do, we use `arg` instead of `param`. See here for example: https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L48-L61